tools/monitors/messaging-monitor/test/messaging.test.ts

```diff
@@ -20,22 +20,29 @@ const SIGNAL_SERVER =
 const PAYLOAD_1: TaggedType<TYPES, 'google.protobuf.Any'> = {
   '@type': 'google.protobuf.Any',
   type_url: 'dxos.Example1',
-  value: Buffer.from('1')
+  value: Buffer.from('1'),
 };
 
 const PAYLOAD_2: TaggedType<TYPES, 'google.protobuf.Any'> = {
   '@type': 'google.protobuf.Any',
   type_url: 'dxos.Example2',
-  value: Buffer.from('2')
+  value: Buffer.from('2'),
 };
 
 const PAYLOAD_3: TaggedType<TYPES, 'google.protobuf.Any'> = {
   '@type': 'google.protobuf.Any',
   type_url: 'dxos.Example3',
-  value: Buffer.from('3')
+  value: Buffer.from('3'),
 };
 
-const createPeer = async (params: Parameters<Messenger['listen']>[0]) => {
+type Peer = {
+  id: PublicKey;
+  signalManager: WebsocketSignalManager;
+  messenger: Messenger;
+  close: () => Promise<void>;
+};
+
+const createPeer = async (params: Parameters<Messenger['listen']>[0]): Promise<Peer> => {
   const signalManager = new WebsocketSignalManager([{ server: SIGNAL_SERVER }]);
```

Comment:
Know you didn't touch this part, but I'll ask anyway: what is the scope of the code/infrastructure under test? Are we testing the interaction of the messaging and protocols packages, the messaging and protocols packages themselves, the functionality of the protocols themselves, the dependent signaling service?

Reply:
Initially this was setup as a part of this tracking issue https://github.com/dxos/dxos/issues/2828. As mentioned before, I think we were hoping this might help fill the gap on some better KUBE signalling monitors/tests, but I don't think that really panned out. In practice, I think this is an integration test to ensure that the latest published version of `@dxos/messaging` works in general and with currently deployed KUBE in particular.